src/app/malaria-101/malaria-101.component.scss

```diff
@@ -0,0 +1,220 @@
+@import "../styles/main";
+
+video {
+    border: 3px solid $dark-grey;
+    background-color: $black !important;
+}
+
+.pieces {
+    height: 125px;
+    width: 115px;
+    border-radius: 25px;
+}
+
+img {
+    max-width: 100%;
+    max-height: 100%;
+    margin: auto;
+}
+
+@mixin text($topPos, $leftPos) {
+    position: absolute;
+    width: 15%;
+    top: $topPos;
+    left: $leftPos;
+}
+
+@mixin rotate($rotation) {
+    -moz-transform: rotate($rotation);
+    -webkit-transform: rotate($rotation);
+    -o-transform: rotate($rotation);
+    -ms-transform: rotate($rotation);
+}
+
+@mixin arrow($topPos, $leftPos, $rotation) {
+    width: 1.5%;
+    height: auto;
+    position: absolute;
+    top: $topPos;
+    left: $leftPos;
+    @include rotate($rotation);    
+}
+
+@mixin drag-drop($topPos, $leftPos) {
+    border-radius: 25px;
+    border: 2px solid $black;
+    width: 9%;
+    height: 19%;
+    position: absolute;
+    top: $topPos;     
+    left: $leftPos; 
+}
+
```

Comment:
This may be the best possible responsiveness for this activity @yukie1412 . I tried different approaches but none of them were as responsive as this.